repo_scanner.py

```diff
@@ -103,23 +108,32 @@ def run_scan(
     """Scan a project with kics."""
 
     logging.info("%s: starting kics scan", project_path)
-    scanner = subprocess.run(
-        [
-            "kics",
-            "scan",
-            "-i",
-            "a88baa34-e2ad-44ea-ad6f-8cac87bc7c71",  # kics id for leaked credential scan only
-            "--no-color",
-            "-p",
-            ".",
-        ],
-        cwd=tmpdir,
-        capture_output=True,
-        check=False,
-    )
+    try:
+        scanner = subprocess.run(
+            [
+                "kics",
+                "scan",
+                "-i",
+                "a88baa34-e2ad-44ea-ad6f-8cac87bc7c71",  # kics id for leaked credential scan only
+                "--no-color",
+                "-p",
+                ".",
+            ],
+            cwd=tmpdir,
+            timeout=60,
+            capture_output=True,
+            check=False,
+        )
 
-    logging.debug(scanner.stdout)
-    logging.debug(scanner.stderr)
+        logging.debug(scanner.stdout)
+        logging.debug(scanner.stderr)
+
+    except subprocess.TimeoutExpired:
+        logging.info("kics scan timed out after 60 seconds - skip repository")
+        scanner = Scanner(
+            returncode = 50,
+            stdout = b"kics scan timed out after 60 seconds - please exclude your repository with label automated-credential-scan-disabled"
```

Comment:
This won't be written to the repo but just into the logs, correct?

Wouldn't it be better to create an issue in the repo saying that the scan failed?
If this does not get fixed, than the scanner could label the repository `automated-credential-scan-disabled` with after some tries (if the permissions allow it).

Reply:
In fact, both happen. 

`logging.info("kics scan timed out after 60 seconds - skip repository")` creates a line in the Gitlab-CI log outout and 
```
scanner = Scanner(
            returncode = 50,
            stdout = b"kics scan timed out after 60 seconds - please exclude your repository with label automated-credential-scan-disabled"
            )
```

creates an issue with the Text in stdout.